terminologie/codesysteme/CodeSystem-KontaktDiagnoseProzedur.json

```diff
@@ -0,0 +1,65 @@
+{
+  "resourceType": "CodeSystem",
+  "status": "draft",
+  "content": "complete",
+  "name": "KontaktDiagnoseProzedur",
+  "id": "KontaktDiagnoseProzedur",
+  "description": "Rolle/Relevanz von Diagnosen undProzeduren im Encounter-Kontext",
+  "version": "1.2.0",
+  "url": "http://example.org/CodeSystem/KontaktDiagnoseProzedur",
+  "concept": [
```

Comment:
Ich wäre dafür, dass wir uns die Mühe machen und weiterführende Links in die Definition der Codes zu packen

Reply:
Haben wir welche...?

Reply:
Nein wäre Rechercheaufwand

Reply:
Issue  aufmachen? In der Hoffnung, dass jemand mal Zeit & Lust hat...? ;-)